components/ai-services-mgt/org.wso2.carbon.ai.service.mgt/pom.xml

```diff
@@ -0,0 +1,201 @@
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.logging</groupId>
+            <artifactId>pax-logging-api</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>
+                            ${project.artifactId}
+                        </Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Import-Package>
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            com.google.gson;version="${com.google.code.gson.osgi.version.range}",
+                            org.wso2.carbon.identity.core.util; version="${carbon.identity.package.import.version.range}",
+                            org.apache.commons.lang; version="${commons-lang.wso2.osgi.version.range}",
+                            org.apache.commons.logging; version="${import.package.version.commons.logging}",
+                            com.fasterxml.jackson.databind.*; version="${com.fasterxml.jackson.annotation.version.range}",
+                            org.wso2.carbon.context; version="${carbon.kernel.package.import.version.range}",
+
+                            org.apache.http; version="${httpcore.version.osgi.import.range}",
+                            org.apache.http.client; version="${httpcomponents-httpclient.imp.pkg.version.range}",
+                            org.apache.http.client.methods; version="${httpcomponents-httpclient.imp.pkg.version.range}",
+                            org.apache.http.entity; version="${httpcore.version.osgi.import.range}",
+                            org.apache.http.message; version="${httpcore.version.osgi.import.range}",
+                            org.apache.http.protocol; version="${httpcore.version.osgi.import.range}",
+                            org.apache.http.util; version="${httpcore.version.osgi.import.range}",
+                            org.apache.http.impl.client; version="${httpcomponents-httpclient.imp.pkg.version.range}",
+                            org.apache.http.impl.nio.client; version="${httpasyncclient.version.osgi.import.range}",
+                            org.apache.http.concurrent; version="${httpcore.version.osgi.import.range}",
+                        </Import-Package>
+                        <Export-Package>
+                            org.wso2.carbon.ai.service.mgt.*; version="${carbon.identity.package.export.version}"
```

Comment:
Export on the necessary packages. Refer https://github.com/wso2-extensions/identity-organization-management/pull/415/files#diff-14b084926843894a0c5ada151f6e58de555097d5fb869d352a4a0c4e75986153R116-R124